oar/core/notification.py

```diff
@@ -239,6 +238,7 @@ def send_email(self, to_addrs, subject, content):
 
 class SlackClient:
     def __init__(self, bot_token):
+        self.cache_dict = dict()
```

Comment:
can we do code formatting, put all the local vars together
```
self.a = xx
self.b = yy
```

Reply:
For me it looks like declaring and initializing the runtime cache variable inside the constructor is fine (though it is referenced in two different methods).  I don't see any difference in moving its declaration to the respective methods. For eg:
```
#Case-1: Declare and initialize the variable in constructor
class test:
    def __init__(self):
        self.attr1 = dict()
    
    def a(self):
        self.attr1 = {1:'one'}
        print(self.attr1) 
    
    def b(self):
        self.attr1 = {2:'two'}
        print(self.attr1)

#Case-2: Declare and initialize the variable in respective methods
class test1:
    def __init__(self):
        print("Inside test1 class")
    
    def aa(self):
        self.attr1 = dict()
        self.attr1 = {1:'one'}
        print(self.attr1) 
    
    def bb(self):
        self.attr1 = dict()
        self.attr1 = {2:'Two'}
        print(self.attr1) 
 
#For Case-1      
t = test()
t.a()  
t.b()

#For Case-2
tt = test1()
tt.aa()
tt.bb()
```
From your comment my understanding is that you are suggesting to go for  Case-2 in the above example.
Though Case-1 and Case-2 are trying to achieve the same thing ,  Case-1 looks more efficient to me.
If my understanding is wrong could you please clarify what exactly are you suggesting?

Reply:
I just want to make the code looks clean, put the all the local var declaration code together. 
```suggestion
				if not bot_token:
            raise NotificationException("slack bot token is not available")
        self.client = WebClient(token=bot_token)
        self.cache_dict = dict()
        
```